nncf/experimental/openvino_native/graph/metatypes/openvino_metatypes.py

```diff
@@ -31,18 +31,42 @@ def get_all_aliases(cls) -> List[str]:
         return cls.op_names
 
 
+@dataclass
+class OpWeightDef:
```

Comment:
I wonder if this is needed at all to keep only one property of operation? If not, maybe it is more reasonable just to keep his inside operation, e.g. to have `weight_channel_axis` within the operation.

Reply:
deleted